site/docs/red-team/strategies/other-encodings.md

```diff
@@ -64,7 +81,7 @@ strategies:
 
 ## Example
 
-Here's a complete example that applies these encoding transformations to test cases:
+Here's a complete example that applies the encoding collection to test cases:
```

Comment:
Prefer the original text as you've redefined "collection" to refer to `other-encodings` whereas the example uses its contents individually.